files/fr/web/html/global_attributes/enterkeyhint/index.html

```diff
@@ -0,0 +1,96 @@
+
+<h3 id="values">Valeurs</h3>
+
+<p>L'attribut <code>enterkeyhint</code> est un attribut à valeur contrainte et accepte uniquement les valeurs suivantes&nbsp;:</p>
+
+<table class="standard-table">
+  <colgroup>
+    <col style="width: 30%;">
+    <col style="width: 50%;">
+    <col style="width: 20%;">
+   </colgroup>
+  <thead>
+   <tr>
+    <th>Valeur</th>
+    <th>Description</th>
+    <th>Libellé de l'exemple<br><small> (dépend de l'agent utilisateur et de la langue)</small></th>
+   </tr>
+  </thead>
+  <tbody>
+   <tr>
+    <td><code>enterkeyhint="enter"</code></td>
+    <td>Généralement utilisé pour insérer une nouvelle ligne de texte.</td>
+    <td><kbd>↵</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="done"</code></td>
+    <td>Signifie généralement qu'il n'y a rien de plus à saisir et que la la méthode de saisie de l'éditeur (IME) est fermée.</td>
+    <td><kbd>Done</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="go"</code></td>
+    <td>Généralement utilisé pour envoyer le visiteur à la cible correspondant au texte saisi.</td>
+    <td><kbd>Open</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="next"</code></td>
+    <td>Cela envoie généralement la personne visitant le site vers le prochain champ qui accepte une saisie de texte.</td>
+    <td><kbd>Next</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="previous"</code></td>
+    <td>Cela envoie généralement la personne visitant le site vers le champ précédent acceptant une saisie de texte.</td>
+    <td><kbd>Prev</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="search"</code></td>
+    <td>Cela envoie généralement la personne visitant le site aux résultats de la recherche pour le texte qui a été saisi.</td>
+    <td><kbd>Search</kbd></td>
+   </tr>
+   <tr>
+    <td><code>enterkeyhint="send"</code></td>
+    <td>Cela envoie généralement le texte sur une cible donnée.</td>
+    <td><kbd>Send</kbd></td>
+   </tr>
+  </tbody>
+</table>
+
+<h2 id="specifications">Spécifications</h2>
+
+<p>{{Specifications}}</p>
+
+<h2 id="browser_compatibility">Compatibilité des navigateurs</h2>
+
+<p>{{Compat}}</p>
+
+<h2 id="see_also">Voir aussi</h2>
+
+<ul>
+  <li>La propriété <a href="/fr/docs/Web/API/HTMLElement/enterkeyhint"><code>HTMLElement.enterkeyhint</code></a> reflétant cet attribut&nbsp;;</li>
+  <li>l'attribut universel <a href="/fr/docs/Web/HTML/Global_attributes/inputmode"><code>inputmode</code></a>&nbsp;;</li>
+  <li>l'attribut universel <a href="/fr/docs/Web/HTML/Global_attributes/contenteditable"><code>contenteditable</code></a>&nbsp;;</li>
+  <li>les attributs <a href="/fr/docs/Web/HTML/Element/input#input_types"><code>type</code></a> et <a href="/fr/docs/Web/HTML/Element/input#htmlattrdefpattern"><code>pattern</code></a> sur les éléments <a href="/fr/docs/Web/HTML/Element/input"><code>&lt;input&gt;</code></a>.</li>
```

Comment:
<subject to debate/non normative/mandatory for this PR>For non "inline" lists (like the ones in "See also") I'd prefer having uppercase for the first letter of each item and no punctuation signs (it's not really a sentence)

Reply:
works for me
fixed in https://github.com/mdn/translated-content/pull/1823/commits/047ee2186aa63fefe5382046869d2d4fcd5338ad